lib/counter_culture/active_record/associations/has_many_association.rb

```diff
@@ -0,0 +1,64 @@
+module CounterCulture
+  module ActiveRecord
+    module Associations
+      module HasManyAssociation
+
+        private
+
+        # Overwrite method of `ActiveRecord::Associations:HasManyAssociation`
+        def count_records
```

Comment:
Is there a way to do this that doesn't override a private method? Could we override the `size` method itself instead?